web-app/src/main/java/com/google/sps/data/Rectangle.java

```diff
@@ -1,31 +1,55 @@
 package com.google.sps.data;
 
 public class Rectangle {
-  double topLeftLat;
-  double topLeftLng;
-  double bottomRightLat;
-  double bottomRightLng;
+  private double topLeftLat;
+  private double topLeftLng;
+  private double bottomRightLat;
+  private double bottomRightLng;
+  private double centerLat;
+  private double centerLng;
 
   Rectangle(double topLeftLat, double topLeftLng, double bottomRightLat, double bottomRightLng) {
     this.topLeftLat = topLeftLat;
     this.topLeftLng = topLeftLng;
     this.bottomRightLat = bottomRightLat;
     this.bottomRightLng = bottomRightLng;
+    this.centerLat = (topLeftLat + bottomRightLat) / 2;
+    this.centerLng = (topLeftLng + bottomRightLng) / 2;
   }
 
-  double getTopLeftLat() {
+  public double getTopLeftLat() {
```

Comment:
I've moved all of these (and the change in print) to a parent PR as you requested, but it's not updating here.